### PR TITLE
Added .doctrine-project.json

### PR DESCRIPTION
**Pull request by TomHAnderson**

### CODE REVIEW

**Comment by TomHAnderson** [comment]
I believe with this PR it is time to drop the master branch.

File: .doctrine-project.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    {
+      "name":"3.x",
+      "branchName":"3.x",
+      "slug":"3.x",
+      "current":true,
+      "maintained":true,
+      "aliases":[
+        "current",
+        "stable",
+        "3.x"
```
Comment (SenseException) [inline .doctrine-project.json]: AFAIK you don't need this one if it is the same value as the slug.

File: .doctrine-project.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      ]
+    },
+    {
+      "name":"release/1.x",
+      "branchName":"release/1.x",
+      "slug":"release/1.x",
+      "maintained":false,
+      "aliases":[
+        "release/1.x"
+      ]
```
Comment (SenseException) [inline .doctrine-project.json]:
```suggestion
      "name":"1.x",
      "branchName":"release/1.x",
      "slug":"1.x",
      "maintained":false
```

**Review by SenseException — CHANGES_REQUESTED** [review]
Are `feature/drop_php5.6` and `feature/stable-doctrine-module` relevant for the documentation?

**Review by greg0ire — APPROVED** [review]

**Comment by SenseException** [comment]
@TomHAnderson Do you want to keep name and slug on `release/1.x`?

**Comment by TomHAnderson** [comment]
@SenseException I don't understand a reason not to keep `release/1.x` as-is.  Are you suggesting removing those properties and just leaving branchName and maintined?

**Comment by SenseException** [comment]
It would affect the rendering of namings on the website. It's a cosmetic change, not a technical one.

**Comment by greg0ire** [comment]
I think @SenseException was suggesting using `1.x` as name and slug, and drop the `release/` part.

**Comment by TomHAnderson** [comment]
Thanks you two.  I've simplified the name and slug to `1.x`

**Pull request merged**